src/intro-to-ens.yaml

```diff
@@ -11,8 +11,336 @@ publishStatus: Live
+            key: C
+          - content: A smarcontract contract can be a resolver That meets relevant conditions for being.Users can create their own resolver
+            key: D
+      
+      - uuid: 310ec5f6-5879-44f9-8306-fb586ccd8dd8
+        content: What is node in ENS?
+        answerKeys:
+          - A
+        type: SingleChoice
+        choices:
+          - content: Node is the output of namshash algorithm with the name as input.
+            key: A
+          - content: Node is an another smart-contract which stores the domains and addresses for backup
+            key: B
+          - content: Node is an algorithm which is used for hashing the names of ENS domains
+            key: C
+          - content: None of these
+            key: D
+      
+  - uuid: 5f64cab6-c18d-4836-b5be-23469c97478d
+    name: Working
+    content: |
+        ### Working 
+
+        Users can buy an available domain, after buying the domain user must set up the addresses for the wallet and resolver.User can set a public resolver or a custom resolver. name is hashed and recorded in the registry, 
+        along with their node's address, resolver address, and TTL. The nodes are mapped to the record, which has the owner’s address, resolver address, and TTL. we need to assign the resolver after owning the domain successfully. 
+        we can set up a public resolver that has most of the necessary functionalities. However, users can also create and use their own custom resolvers according to their specific needs. Users are also required to setup reverse 
+        records for replacing of addresses with domain names such that names are displayed instead of addresses.
+
+        Resolving a name in ENS is a two-step process: First, ask the registry what resolver is responsible for the name, and second, ask that resolver for the answer to your query. The domain is owned by the owner till the deadline 
+        after that the owner has to renew it within 90 days' grace time. After the grace period, the name is released for registration by anyone with a temporary premium which decreases over a 28 days period to prevent domain squatters.
+        The released and registered names can be viewed in [Dune analytics dashboard](https://dune.com/makoto/ens-released-to-be-released-names).Anyone can extend a domain’s expiration period but the ownership is still maintained by the owner 
+        of that domain.Currently registration and renewal cost $5/year for names that are five characters or longer. Four character names cost $160/year, and three character names cost $640/year. Domains are bought using Eth similarly gas fees are paid in Eth.
+        
+        ### ENS as NFTs 
+
+        Ens domains with .eth are NFTs because they are in ERC-721 standards.All NFTs have a unique uint256 tokenID from which we can convert to the actual content or art or data. The domains with .eth are traded and exchanged like NFTs.The tokenID’s are globally unique.
```

Comment:
```suggestion
        ENS domains with .eth are NFTs because they are in ERC-721 standards. All NFTs have a unique uint256 tokenID from which we can convert to the actual content or art or data. The domains with .eth are traded and exchanged like NFTs. The tokenID’s are globally unique.
```